src/client/components/Tooltip/TooltipIcon.tsx

```diff
@@ -31,10 +32,16 @@ export const TooltipIcon = (props: TooltipIconProps) => {
         onMouseLeave={() => setIsVisible(false)}
         onTouchStart={() => setIsVisible(true)}
       >
-        <InfoIcon
-          size={ICON_SIZE}
-          color={isVisible ? colorsV3.gray700 : colorsV3.gray900}
-        />
+        {filled ? (
+          <InfoIconFilled
```

Comment:
Shouldn't `size={ICON_SIZE}` be provided for `<InfoIconFilled />` as well? 🤔

Reply:
Good catch! :D Thank you!